service/operator/operatorkit.go

```diff
@@ -1,145 +0,0 @@
-package operator
-
-import (
-	microerror "github.com/giantswarm/microkit/error"
-)
-
-// Operator implements the building blocks of any operator business logic being
-// reconciled when observing TPRs. This interface provides a guideline for an
-// easier way to follow the rather complex intentions of operators in general.
-type Operator interface {
```

Comment:
I was debugging and deploying back and forth and wondered why the changes in the vendored dependency are not applied. Well, reason was the operator was still using this code here. I wasted two hours finding this out. Great.